pkg/mflag/flag.go

```diff
@@ -557,21 +578,30 @@ func (fs *FlagSet) PrintDefaults() {
 				names = append(names, name)
 			}
 		}
+
 		if len(names) > 0 && len(flag.Usage) > 0 {
-			val := flag.DefValue
 
-			if home != "" && strings.HasPrefix(val, home) {
-				val = homedir.GetShortcutString() + val[len(home):]
+			nstring := fmt.Sprintf("-%s", strings.Join(names, ", -"))
+			if len(names) == 1 && strings.HasPrefix(names[0], "-") {
+				nstring = "    " + nstring
 			}
 
-			if isZeroValue(val) {
-				format := "  -%s"
-				fmt.Fprintf(writer, format, strings.Join(names, ", -"))
+			if isZeroValue(flag.DefValue) && flag.Value.Type() == "BOOL" {
+				fmt.Fprintf(writer, "  %s", nstring)
 			} else {
-				format := "  -%s=%s"
-				fmt.Fprintf(writer, format, strings.Join(names, ", -"), val)
+				fmt.Fprintf(writer, "  %s=%s", nstring, flag.Value.Type())
+			}
+
+			usage := flag.Usage
+			if !isZeroValue(flag.DefValue) {
+				val := flag.DefValue
+				if home != "" && strings.HasPrefix(val, home) {
+					val = homedir.GetShortcutString() + val[len(home):]
+				}
+
+				usage = fmt.Sprintf("%s, default is '%s'", usage, val)
```

Comment:
I tried a variety of string formats for displaying the default value, and `default is 'VAL'` was the easiest to read for me, especially when trying to quickly discern a default value.  Using a shorter string, e.g. `default='VAL'`, would allow the CLI help test max width constants to shrink to 80/100 chars (currently 83/103), but readability of the output seemed more important than nice, round constants.